js/src/models/model.layers.js

```diff
@@ -553,8 +553,17 @@ Wu.Model.Layer = Wu.Model.extend({
 	},
 
 	_save : function (json) {
-		var string  = JSON.stringify(json);
-		Wu.save('/api/layer/update', string);
+		app.api.updateLayer(json, function (err, result) {
+			if (err) console.error('err', err);
```

Comment:
better error handling here - give user feedback if something went wrong.

`app.feedback.setError()` (don't quite remember correct syntax, but look in `/js/src/panes/pane.feedback.js`